src/lib/components/Header/Header.svelte

```diff
@@ -41,12 +38,9 @@
       </button>
 
       <a class="ldaf-logo__compact" href="/">
-        <Logo />
+        <span class="usa-sr-only">Go to home page.</span>
```

Comment:
Previously the `alt` text for the old logo was "Louisiana Department of Agriculture and Forestry Home", which wasn't quite accurate since it wasn't always used as a link. Now the `alt` text just informs users that this image is a logo, and we use this pattern when the logo is wrapped in a link to inform users of where the link goes.